tests/bluechi_test/machine.py

```diff
@@ -122,6 +122,22 @@ def copy_container_script(self, script_file_name: str):
              with content:\n{content}")
         self.create_file(target_dir, script_file_name, content)
 
+    def copy_machine_lib(self):
+        LOGGER.info("Copying machine lib...")
+        curr_dir = os.getcwd()
+        source_dir = os.path.join(curr_dir, "..", "..", "..", "bluechi_test", "bluechilib")
```

Comment:
Assuming the lib would be in `tests/bluechi_machine_lib` we could also do:
```suggestion
        import bluechi_machine_lib as bml
        source_dir = bml.__path__[0]
        # ...
        # 
        target_dir = os.path.join("/", "tmp", bml.__name__)
```
Then we don't need any relative paths and we are sure that the module (`bluechi_machine_lib` in that case) exists.

Reply:
Nice

Reply:
Done